raiden/api/rest.py

```diff
@@ -957,7 +958,7 @@ def initiate_payment(
                 amount=amount,
                 identifier=identifier,
             )
-        except (InvalidAmount, InvalidAddress, UnknownTokenAddress) as e:
+        except (InvalidAmount, InvalidAddress, PaymentConflict, UnknownTokenAddress) as e:
```

Comment:
Please add a test checking this exception gets raised and that we return the proper response.